container/dependencies.nix

```diff
@@ -28,8 +37,8 @@ let patched_rPackages =
   pandoc quarto
   dcm2niix dcmtk gdcm simpleitk ants
   cudaPackages.cudatoolkit
-  snakemake
-  (nixgl.nvidiaPackages { version = "470.103.01"; sha256 = "19c7r3nrdi48vkzg6ykj7hfkgmvm49xhydj61hxlc4y2i6gk1hjn"; }).nixGLNvidia
+  snakemake ] ++ glWrappers ++ [
```

Comment:
is there any point having more `glWrappers` than we have `nixGL` supporting?

Reply:
I don't know what you mean, glWrappers are generated from the list of supported drivers